docs/api/qiskit/0.46/transpiler_preset.mdx

```diff
@@ -29,7 +29,7 @@ This module contains functions for generating the preset pass managers for the t
 <Function id="qiskit.transpiler.preset_passmanagers.generate_preset_pass_manager" github="https://github.com/qiskit/qiskit/tree/stable/0.46/qiskit/transpiler/preset_passmanagers/__init__.py" signature="qiskit.transpiler.preset_passmanagers.generate_preset_pass_manager(optimization_level, backend=None, target=None, basis_gates=None, inst_map=None, coupling_map=None, instruction_durations=None, backend_properties=None, timing_constraints=None, initial_layout=None, layout_method=None, routing_method=None, translation_method=None, scheduling_method=None, approximation_degree=None, seed_transpiler=None, unitary_synthesis_method='default', unitary_synthesis_plugin_config=None, hls_config=None, init_method=None, optimization_method=None, *, _skip_target=False)">
   Generate a preset [`PassManager`](qiskit.transpiler.PassManager "qiskit.transpiler.PassManager")
 
-  This function is used to quickly generate a preset pass manager. A preset pass manager are the default pass managers used by the [`transpile()`](compiler#qiskit.compiler.transpile "qiskit.compiler.transpile") function. This function provides a convenient and simple method to construct a standalone [`PassManager`](qiskit.transpiler.PassManager "qiskit.transpiler.PassManager") object that mirrors what the [`transpile()`](compiler#qiskit.compiler.transpile "qiskit.compiler.transpile") function internally builds and uses.
+  This function is used to quickly generate a preset pass manager. Preset pass managers are the default pass managers used by the [`transpile()`](compiler#qiskit.compiler.transpile "qiskit.compiler.transpile") function. This function provides a convenient and simple method to construct a standalone [`PassManager`](qiskit.transpiler.PassManager "qiskit.transpiler.PassManager") object that mirrors what the [`transpile()`](compiler#qiskit.compiler.transpile "qiskit.compiler.transpile") function internally builds and uses.
```

Comment:
@Eric-Arellano

I manually updated the HTML for 0.46 and 1.1 as well to fix this line that I'm assuming is a typo in `stable/0.46` and `stable/1.1`.

As you can see, the [dev](https://github.com/Qiskit/qiskit/blob/main/qiskit/transpiler/preset_passmanagers/__init__.py#L109) version uses plural (Preset pass managers) compared to singular (A preset pass manager) for [0.46](https://github.com/Qiskit/qiskit/blob/stable/0.46/qiskit/transpiler/preset_passmanagers/__init__.py#L98) and [1.1](https://github.com/Qiskit/qiskit/blob/stable/1.1/qiskit/transpiler/preset_passmanagers/__init__.py#L99).

I can also create a PR in the qiskit repo to update it afterward :+1:

Reply:
> I can also create a PR in the qiskit repo to update it afterward 👍

Yes please! It looks like they will have another 0.46 and 1.1 release https://github.com/Qiskit/qiskit/milestones